### ISSUE TITLE
Feed lists not working

### ISSUE THREAD

**Issue by licjon** [skeeto/elfeed#418]
I'm using Doom Emacs. My elfeed-feeds was not working, so I added some feeds with elfeed-add-feed. I don't know where these URLs are stored. I replaced elfeed-feeds list with elfeed-org, but I still only get the feeds I added with elfeed-add-feed. Is there a way to only get the feeds on my list? Is there a way to access the file that stores the URLs added by elfeed-add-feed?

**Comment by licjon** [comment]
I think it has something to do with the database, which I do not know how to find or configure. I tried adding (setq elfeed-db-directory "~/.emacs.d/elfeed.org") to my config. It basically broke everything, but when I removed it, it did succeed in reseting the feeds so that the ones I added with elfeed-add-feed were gone. I deleted elfeed-org from my config, and elfeed updated with what was in my elfeed-feeds list, but editing this list has no effect once again.
